Paintroid/src/main/java/org/catrobat/paintroid/MainActivity.java

```diff
@@ -436,15 +436,20 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
 	}
 
 	@Override
-	public void onRequestPermissionsResult(int requestCode, String[] permissions, int[] grantResults) {
-		presenter.handlePermissionRequestResults(requestCode, permissions, grantResults);
+	public void onRequestPermissionsResult(int requestCode, @NonNull String[] permissions, @NonNull int[] grantResults) {
+		presenter.handleRequestPermissionResult(requestCode, permissions, grantResults);
 	}
 
 	@Override
-	public void forwardActivityResult(int requestCode, int resultCode, Intent data) {
+	public void baseActivityResult(int requestCode, int resultCode, Intent data) {
 		super.onActivityResult(requestCode, resultCode, data);
 	}
 
+	@Override
+	public void baseHandlePermissionRequestResults(int requestCode, @NonNull String[] permissions, @NonNull int[] grantResults) {
```

Comment:
analogue to  `baseActivityResult(..)` you could consider using the term `super` instead of `base` here to clarify what it does.

Reply:
Thanks, fixed.